Makefile

```diff
@@ -1,104 +1,38 @@
-.PHONY: rel deps test show_test_results
+.PHONY: rel
 LOG=$(subst TARGET,$@,TARGET.log 2>&1 || (cat TARGET.log; exit 1))
 SILENCE_COVER =  | grep -v "logs.*\\.coverdata"
 SILENCE_COVER += | grep -v "Analysis includes data from imported files"
 SILENCE_COVER += | grep -v "WARNING: Deleting data for module"
 LOG_SILENCE_COVER=$(subst TARGET,$@,TARGET.log 2>&1 || (cat TARGET.log $(SILENCE_COVER); exit 1))
 EJABBERD_DIR = apps/ejabberd
-EJD_INCLUDE = $(EJABBERD_DIR)/include
-EJD_PRIV = $(EJABBERD_DIR)/priv
 XEP_TOOL = tools/xep_tool
 EJD_EBIN = $(EJABBERD_DIR)/ebin
-DEVNODES = node1 node2 node3 fed1
-
-all: deps compile
-
-compile: rebar
-	./rebar $(OPTS) compile > $(LOG)
-
-deps: rebar
-	./rebar get-deps > $(LOG)
-
-clean: rebar configure.out
-	rm -rf apps/*/logs
-	. ./configure.out && ./rebar clean
-
-quick_compile: rebar
-	./rebar $(OPTS) compile skip_deps=true > $(LOG)
-
-reload: quick_compile
-	@E=`ls ./rel/mongooseim/lib/ | grep ejabberd-2 | sort -r | head -n 1` ;\
-	rsync -uW ./apps/ejabberd/ebin/*beam ./rel/mongooseim/lib/$$E/ebin/ ;\
-
-reload_dev: quick_compile
```

Comment:
`./rebar3 as mim1,mim2,mim3,fed1 compile` creates build inside _build with options merged from all specified profiles. This will not work for just recompiling code and putting in relevant builds.